Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js

```diff
@@ -111,7 +147,8 @@ YUI.add('ez-searchplugin-tests', function (Y) {
             this.view.fire('contentSearch', {
                 viewName: this.viewName,
                 search: {
-                    criteria: {},
+                    offset: 42,
```

Comment:
do you need to add `offset` and `limit` ? those parameters are documented as optional, so this should not be needed